src/ui.c

```diff
@@ -45,7 +50,7 @@ start_curses()
 	int fd;
 
 	signal(SIGINT, int_handler);
-	freopen("/dev/tty", "r", stdin);
+	assert(freopen("/dev/tty", "r", stdin));
```

Comment:
Would using an `if` statement to check the return value and `err` to error if `freopen` returns `NULL` improve consistency with other parts of the code base?